components/colorwheel/index.css

```
@@ -56,35 +94,68 @@
   }
 }
 
-.spectrum-ColorWheel-gradient {
-  position: relative;
-  z-index: 0;
-  width: var(--spectrum-colorwheel-width);
-  height: var(--spectrum-colorwheel-height);
-  border-radius: 100%;
-  box-sizing: border-box;
-
-  border-width: var(--spectrum-colorwheel-border-size);
-  border-style: solid;
+.spectrum-ColorWheel-internal{
+  display: flex;
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  width: var(--spectrum-colorwheel-colorarea-container-size);
+  height: var(--spectrum-colorwheel-colorarea-container-size);
+  margin: auto;
 }
 
-.spectrum-ColorWheel-wheel {
-  position: relative;
-  z-index: 1;
-}
-
-.spectrum-ColorWheel-outerCircle,
-.spectrum-ColorWheel-innerCircle {
-  fill: transparent;
-
-  stroke-width: var(--spectrum-colorwheel-border-size);
+.spectrum-ColorWheel-colorarea-container{
+  height: auto;
+  width: 100%;
+  margin: var(--mod-colorwheel-colorarea-margin, var(--spectrum-colorwheel-colorarea-margin));
 }
 
 .spectrum-ColorWheel-slider {
   @extend %spectrum-ColorControl-hiddenField;
 }
 
 .spectrum-ColorWheel-handle {
+  transform: translate(var(--spectrum-colorwheel-colorhandle-position), 0);
   left: 50%;
   top: 50%;
 }
+
+/* clip path set border-width wider than than spectrum-colorwheel-wheel to create appreance of a border*/
+.spectrum-ColorWheel-border{
+  position: relative;
+  background-color: var(--mod-colorwheel-border-color, var(--spectrum-colorwheel-border-color));
```

Should we also provide a `--highcontrast-` prefixed custom property here? We've been trying to do that for everything that involves color-related properties/values.

```suggestion
  background-color: var(--highcontrast-colorwheel-border-color, var(--mod-colorwheel-border-color, var(--spectrum-colorwheel-border-color)));
```

---

I have questions on this one. Generally our pattern in the code base currently is to not use a high contrast borders in WHCM for the non disabled variants. I was using[ ColorSlider](https://opensource.adobe.com/spectrum-css/colorslider.html) as my example which has a similar "border" and have implemented ColorArea similarly but maybe this is not what we want moving forward. 

Should I change both ColorWheel and ColorArea (both of which do not directly receive focus, ColorHandle does) to have high contrast borders when in WHCM and if so what system color do we want for them?

---

Oh, you may be correct here. I'm actually curious as to what this should look like in WHCM overall.

---

We chatted with Accessibility team in slack, https://adobedesign.slack.com/archives/CSNDSSA5D/p1676488870249269, and have decided on not adding a high contrast border in this case.